modules/hal_nordic/nrfx/nrfx_config.h

```diff
@@ -1042,6 +1039,14 @@
     #include <nrfx_config_nrf5340_application.h>
 #elif defined(NRF5340_XXAA_NETWORK)
     #include <nrfx_config_nrf5340_network.h>
+#elif defined(NRF54H20_ENGB_XXAA) && defined(NRF_APPLICATION)
+    #include <nrfx_config_nrf54h20_engb_application.h>
+#elif defined(NRF54H20_ENGB_XXAA) && defined(NRF_RADIOCORE)
+    #include <nrfx_config_nrf54h20_engb_radiocore.h>
+#elif defined(NRF54H20_ENGB_XXAA) && defined(NRF_PPR)
+    #include <nrfx_config_nrf54h20_engb_ppr.h>
+#elif defined(NRF54H20_ENGB_XXAA) && defined(NRF_FLPR)
+    #include <nrfx_config_nrf54h20_engb_flpr.h>
```

Comment:
Do we actually need a second copy of these config files? Why not just use:
```
#elif (defined(NRF54H20_XXAA) || defined(NRF54H20_ENGB_XXAA)) && defined(NRF_APPLICATION)
    #include <nrfx_config_nrf54h20_application.h>
```
and so on?

Reply:
@gmarull yes, nrfx_config_* for these two targets are identical

Reply:
adjusted